appdaemon/plugins/hass/hassplugin.py

```diff
@@ -446,24 +442,14 @@ def utility(self):
     @hass_check
     async def set_plugin_state(self, namespace, entity_id, **kwargs):
         self.logger.debug("set_plugin_state() %s %s %s", namespace, entity_id, kwargs)
-        config = (await self.AD.plugins.get_plugin_object(namespace)).config
 
-        # TODO cert_path is not used
-        if "cert_path" in config:
-            cert_path = config["cert_path"]
-        else:
-            cert_path = False  # noqa: F841
+        # if we get a request for not our namespace something has gone very wrong
+        assert namespace == self.namespace
```

Comment:
I left these as an assert because iiuc we should really never get a mismatch or that indicates a serious logic error.  If there's a more appropriate exception to throw I'm happy to use another.